Sandbox/conan-recipe/test_package/conanfile.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class NmosCppTestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
```

Comment:
the CONFIG is installed https://github.com/prince-chrismc/nmos-cpp/pull/1/files#diff-a11e7c45055c0be3e4185a7eb8765fe1c2664e9374dada0e19a2068b18eb8fbcR18

```suggestion
    generators = "cmake", "cmake_find_package_multi"
```

Reply:
Ah, OK, makes sense. So different rationale to the main conanfile using those generators!